ethcore/src/trace/types/trace.rs

```diff
@@ -18,19 +18,59 @@
 
 use ethereum_types::{U256, Address, Bloom, BloomInput};
 use bytes::Bytes;
-use rlp::{Rlp, RlpStream, Encodable, DecoderError, Decodable};
+use rlp::{self, Rlp, RlpStream, Encodable, DecoderError, Decodable};
 
 use vm::ActionParams;
 use evm::CallType;
 use super::error::Error;
 
 /// `Call` result.
-#[derive(Debug, Clone, PartialEq, Default, RlpEncodable, RlpDecodable)]
+#[derive(Debug, Clone, PartialEq)]
 pub struct CallResult {
 	/// Gas used by call.
 	pub gas_used: U256,
 	/// Call Output.
-	pub output: Bytes,
+	pub output: Option<Bytes>,
```

Comment:
Is it worth changing the type here? It introduces additional complexity (custom RLP serialization or database migration) and will only be used when tracing calls to builtins - a feature that was not available before at all. @debris what's your thoughts on this?